examples/replica_set_single_disk/values.yaml

```diff
@@ -83,3 +83,22 @@ additionalUsers:
         role: "readWriteAnyDatabase"
       - db: admin
         role: "userAdminAnyDatabase"
+        
+backup:
+  mode: enabled
+  autoTerminateOnDeletion: false
+  encryption:
+    kmip:
+      client:
```

Comment:
This might be a bad idea as MEKO turns KMIP on and off based on the existence of the `.spec.encryption.kmip`. Putting an empty element here might lead to errors.

Reply:
Added validation to verify whether encryption is enabled or not in the charts/templates/mongodb-deployment.yaml file.